src/app/service/categories/ServiceCategories.jsx

```diff
@@ -107,7 +107,7 @@ class ServiceCategories extends Component {
 }
 
 const mapStateToProps = state => ({
-  taxonomy: state.db.taxonomy,
+  taxonomy: state.locations.taxonomy,   //TODO: refactor this so that taxonomy is in its own top-level prop
```

Comment:
I very much agree with this comment, though the one question for me would be: until we do take taxonomy out, is there even a point renaming this to "locations"? As long as taxonomy is under it, calling it "locations" seems more confusing to me than just "db".

Reply:
I'm going to do this refactoring next.